### PR TITLE
DM-46356: Create new verification pipelines for IsrTaskLSST for LATISS

### PR DESCRIPTION
**Pull request by erykoff**

### CODE REVIEW

File: pipelines/_ingredients/verifyBiasLSST.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+description: cp_verify BIAS calibration verification (LSST style).
```
Comment (czwa) [inline pipelines/_ingredients/verifyBiasLSST.yaml]: This ticket merges after DM-45856, correct?  Are these on this ticket as well so they could have parallel development?

Comment (czwa) [reply pipelines/_ingredients/verifyBiasLSST.yaml]: Same ordering comment as on DM-45856.

File: tests/test_pipelines.py [hunk 2]
```diff
@@ -121,7 +122,13 @@ def test_cameras(self):
...
     @unittest.skipIf(not has_obs_lsst, reason="Cannot test LATISS pipelines without obs_lsst")
     def test_latiss_pipelines(self):
-        for pipeline in self._get_pipelines():
+        for pipeline in self._get_pipelines(exclude=[
+                # The following tasks are not part of the new pipelines.
+                "verifyDefectsIndividual.yaml",
+                # The following tasks will be added in the future.
+                "verifyCrosstalk.yaml",
+                "verifyBfk.yaml",
+        ]):
```
Comment (czwa) [inline tests/test_pipelines.py:141]: If I'm understanding this correctly, we will not be testing the legacy pipelines anymore.  Should we keep testing them for now, and deprecate that in the future when we've finished the transition?

**Review by czwa — APPROVED** [review]
I think most of this was covered on the other ticket.  One question on testing legacy pipelines for the transition period.

File: pipelines/_ingredients/verifyBiasLSST.yaml (hunk 1 above)
Comment (erykoff) [reply pipelines/_ingredients/verifyBiasLSST.yaml]: Yup, we can only test the LATISS pipelines on top of the _ingredients in DM-45856.

File: tests/test_pipelines.py (hunk 2 above)
Comment (erykoff) [reply tests/test_pipelines.py:141]: The verifyBfk pipeline will be coming on this ticket.  I don't know if you're referring to the verification pipeline (which should be easy to adapt for crosstalk, I hope, even if it's verifying existing crosstalk!) or the construction pipeline (which is already in rough shape).

**Pull request merged**